src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -22,10 +24,15 @@
 (def ReportersConfig
   {(schema/optional-key :jmx) JmxReporterConfig})
 
+(def JolokiaApiConfig
+  {(schema/optional-key :enabled) schema/Bool
+   (schema/optional-key :config) jolokia/JolokiaConfig})
+
 (def MetricsConfig
   {:server-id                       schema/Str
    (schema/optional-key :enabled)   schema/Bool
-   (schema/optional-key :reporters) ReportersConfig})
+   (schema/optional-key :reporters) ReportersConfig
+   (schema/optional-key :jolokia) JolokiaApiConfig})
```

Comment:
Nevermind! I read this totally wrong - this isn't in the `reporters` section, so TK-393 doesn't matter at all for this!